todo-app/src/Components/todoList.js

```diff
@@ -0,0 +1,57 @@
+//jshint esnext: true
+
+import React from 'react';
+
+import TodoItem from './todoItem';
+import todosData from './todos.json';
+import './styles/todoList.css';
+
+const todos = todosData.todos;
+
+
+export default class TodoList extends React.Component {
+
+    state = {
+        todos,
+    }
+
+    handleIsItDone = (todoItem) => {
+       this.setState({
+           todos: this.state.todos.map(item => {
+               if (item.id === todoItem.id){
+                   item.done = !item.done
+               }
+               return item;
+           })
+       })
+    }
+
+
+    render() {
+        return (
+            <ul className="todosContainer">
+                {this.state.todos.length === 0 ? //is it empty?
+                    <h1>No items to show</h1>
+                    :
+                    this.renderAllTodos()}
+            </ul>
+        )
+    }
+    
+    renderAllTodos = () => {
```

Comment:
Not sure if it was us discussing this last week, but typically devs will put the `render` function last inside React components. Again it's a matter of preference, but you can check out this [eslint rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md) to see some reasoning for this convention.